docs/content/getting-started.md

```diff
@@ -6,32 +6,54 @@ menu: main
 weight: -5
 ---
 
-## Goal
+This tutorial will take you through the process of building a GraphQL server with gqlgen that can:
 
-The aim for this tutorial is to build a "todo" graphql server that can:
-
- - get a list of all todos
- - create new todos
- - mark off todos as they are completed
+ - Return a list of todos
+ - Create new todos
+ - Mark off todos as they are completed
 
 You can find the finished code for this tutorial [here](https://github.com/vektah/gqlgen-tutorials/tree/master/gettingstarted)
 
 ## Install gqlgen
 
-Assuming you already have a working [go environment](https://golang.org/doc/install) you can simply go get:
+This article uses [`dep`](https://github.com/golang/dep) to install gqlgen.  [Follow the instructions for your environment](https://github.com/golang/dep) to install.
+
+Assuming you already have a working [Go environment](https://golang.org/doc/install), create a directory for the project in your `$GOPATH`:
+
+```sh
+$ mkdir -p $GOPATH/src/github.com/[username]/gqlgen-todos
+```
+
+Add the following file to your project under `scripts/gqlgen.go`:
+
+```go
+// +build ignore
```

Comment:
there needs to be an empty line between this comment and package main for the build tag to work